lib/sites.ts

```diff
@@ -52,6 +53,11 @@ export function useSiteData(siteUUID: string) {
     actualsFetcher
   );
 
+  const { data: inverterData, error: inverterError } = useSWR(
```

Comment:
Since you're not using a custom fetcher here anymore, you can tell swr to set the type of `data` inline:
```suggestion
  const { data: inverterData, error: inverterError } = useSWR<Inverters>(
```